### DISCUSSION TITLE
The method HtmlSanitizerConfig::allowedLinkSchemes() generates a Call to undefined method

### DISCUSSION THREAD

**Discussion by daniele-vigano** [symfony/symfony#58577] [Q&A] [answered]
In version 7.1, the call to the method HtmlSanitizerConfig::allowedLinkSchemes(), documented at https://symfony.com/doc/current/html_sanitizer.html#force-allow-link-urls, generates a "Call to undefined method" exception.

I can't understand why; what I am missing? I am using the code from the example in my test code:
```
$postSanitizer = new HtmlSanitizer(
    (new HtmlSanitizerConfig())
        ->forceHttpsUrls()
        ->allowedLinkSchemes(['http', 'https', 'mailto'])
        ->allowedLinkHosts(['symfony.com'])
        ->allowRelativeLinks()
);
```

I'm stuck, help needed.
Thanks

**Comment by stof** [comment, marked as answer]
there is a mistake in the documentation. The methods are named `allowLinkSchemes` and `allowLinkHosts`

**Reply by daniele-vigano** [reply]
Thank you!
